localization/src/main/kotlin/edu/artic/localization/LanguageSelector.kt

```diff
@@ -26,6 +26,15 @@ class LanguageSelector(private val prefs: LocalizationPreferences) {
         }
     }
 
+    fun setTourLanguage(lang: Locale) {
```

Comment:
Rename `lang` to something that makes the `hasNoLanguage()` call read better. Maybe `proposed`?

Reply:
Renamed it to `setTourLanguage`.